### PR TITLE
Adding django-bootstrap5

### PR DESCRIPTION
**Pull request by swainn**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by swainn** [comment]
I am willing to be a maintainer

**Comment by swainn** [comment]
@conda-forge/help-python-c this is good to go

File: recipes/django-bootstrap5/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```
Comment (beenje) [inline recipes/django-bootstrap5/LICENSE.txt:1]: The LICENSE file is part of the pypi package. You can remove it from the recipe.

File: recipes/django-bootstrap5/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/zostera/django-bootstrap5
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```
Comment (beenje) [inline recipes/django-bootstrap5/meta.yaml:38]: The file part of the pypi package is `LICENSE`:
```suggestion
  license_file: LICENSE
```

File: recipes/django-bootstrap5/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.8
```
Comment (beenje) [inline recipes/django-bootstrap5/meta.yaml:22]: I think you could add `importlib-metadata` as run requirements if you want to support python 3.6 and 3.7.
But not sure not sure why there is a constraint to `<3` in the `setup.py`...

If you are happy with `>=3.8` (3.6 is soon EOL), I'd put the same constraint for the host python requirement.

**Review by beenje — CHANGES_REQUESTED** [review]

File: recipes/django-bootstrap5/LICENSE.txt (hunk 1 above)
Comment (swainn) [reply recipes/django-bootstrap5/LICENSE.txt]: I've removed the license file from the recipe.

File: recipes/django-bootstrap5/meta.yaml (hunk 3 above)
Comment (swainn) [reply recipes/django-bootstrap5/meta.yaml:22]: I'm happy with the 3.8 requirements. Updated host requirement as suggested.

**Comment by swainn** [comment]
@beenje I've addressed your feedback.

**Comment by beenje** [comment]
Thanks.

@conda-forge/help-python this recipe looks good to me

**Review by dopplershift — APPROVED** [review]
Ignoring the Windows build failure for noarch (which is due to conda-forge/django-feedstock#132)

**Pull request merged**